src/backend/metal/src/command.rs

```diff
@@ -1976,14 +1969,28 @@ impl com::RawCommandBuffer<Backend> for CommandBuffer {
         T: IntoIterator,
         T::Item: Borrow<com::ImageCopy>,
     {
+        let inner = unsafe { &mut *self.inner.get() };
```

Comment:
Maybe we should just have a function to do this? We have this a few times now

Reply:
uh, I approached this and now regret it - the changes are blowing out of control
perhaps, we can proceed and I'll follow up?